### PR TITLE
Support ingest multiple files in one raft command

### PR DESCRIPTION
**Pull request by Little-Wallace**
Signed-off-by: Little-Wallace <bupt2013211450@gmail.com>

### What problem does this PR solve?

Issue Number: part of https://github.com/pingcap/tics/issues/1811

Problem Summary:
Tiflash need to transfer sst to tiflash-format file and if lightning just retried ingesting a write-cf file but it did not retry ingesting a default-cf, tiflash will go panic. 

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- PR to update `pingcap/tidb-ansible`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 
- Support ingesting default-cf and write-cf sst in one raft command.

### CODE REVIEW

File: src/import/sst_service.rs [hunk 1]
```diff
@@ -94,6 +96,100 @@ where
...
+                cmd.mut_requests().push(ingest);
+            }
+            cmd.mut_header().set_term(resp_header.get_current_term());
+
+            // Here we shall check whether the file has been ingested before. This operation
+            // must execute after geting a snapshot from raftstore to make sure that the
+            // current leader has applied to current term.
+            for sst in ssts.iter() {
+                if !importer.exist(&sst) {
+                    warn!(
```
Comment (glorv) [inline src/import/sst_service.rs:170]: Is it better to return an error than just ignore it?

Comment (Little-Wallace) [reply src/import/sst_service.rs:170]: No.... If I returned an error, lightning may retry again and again......

Comment (glorv) [reply src/import/sst_service.rs:170]: Why, if the sst file does not exist, lightning will just write another, this shouldn't cause dead lock

**Review by 3pointer — COMMENTED** [review]
/lgtm

**Pull request merged**